src/main/java/spoon/support/reflect/declaration/CtElementImpl.java

```diff
@@ -328,14 +328,14 @@ private boolean compare(Object o1, Object o2) {
 		return o1 == o2;
 	}
 
-	public void replace(Filter<? extends CtElement> replacementPoints,
-			CtElement element) {
-		List<? extends CtElement> l = Query
-				.getElements(this, replacementPoints);
-		for (CtElement e : l) {
-			e.replace(element);
-		}
-	}
+//	public void replace(Filter<? extends CtElement> replacementPoints,
+//			CtElement element) {
+//		List<? extends CtElement> l = Query
+//				.getElements(this, replacementPoints);
+//		for (CtElement e : l) {
+//			e.replace(element);
+//		}
+//	}
```

Comment:
If it is no more used, remove this method. We have the git history if we would like to retrieve something.